dependencies/maven/artifacts.bzl

```diff
@@ -1,45 +0,0 @@
-#
-# Copyright (C) 2022 Vaticle
-#
-# Licensed to the Apache Software Foundation (ASF) under one
-# or more contributor license agreements.  See the NOTICE file
-# distributed with this work for additional information
-# regarding copyright ownership.  The ASF licenses this file
-# to you under the Apache License, Version 2.0 (the
-# "License"); you may not use this file except in compliance
-# with the License.  You may obtain a copy of the License at
-#
-#   http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing,
-# software distributed under the License is distributed on an
-# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-# KIND, either express or implied.  See the License for the
-# specific language governing permissions and limitations
-# under the License.
-#
-
-artifacts = [
-    "com.google.code.gson:gson",
-    "com.univocity:univocity-parsers",
-    "commons-io:commons-io",
-    "info.picocli:picocli",
-    "io.cucumber:cucumber-java",
-    "io.cucumber:cucumber-junit",
-    "javax.xml.stream:stax-api",
-    "junit:junit",
-    "org.jetbrains.compose.foundation:foundation-desktop",
-    "org.jetbrains.compose.foundation:foundation-layout-desktop",
-    "org.jetbrains.compose.material:material-desktop",
-    "org.jetbrains.compose.runtime:runtime-desktop",
-    "org.jetbrains.compose.ui:ui-desktop",
-    "org.jetbrains.compose.ui:ui-graphics-desktop",
-    "org.jetbrains.compose.ui:ui-text-desktop",
-    "org.jetbrains.compose.ui:ui-unit-desktop",
-    "org.jetbrains.kotlinx:kotlinx-coroutines-core-jvm",
-    "org.jetbrains.skiko:skiko-awt-runtime-linux-x64",
-    "org.jetbrains.skiko:skiko-awt-runtime-macos-x64",
-    "org.jetbrains.skiko:skiko-awt-runtime-windows-x64",
-    "org.kohsuke:github-api",
-    "org.sharegov:mjson",
```

Comment:
Let's do a clean up here. You only use the json one so that should be all we need here.